src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -434,7 +434,7 @@ class CLISecurityDomainOperation(str, Enum):
         with self.argument_context('keyvault key {}'.format(item)) as c:
             c.argument('protection', arg_type=get_enum_type(['software', 'hsm']), options_list=['--protection', '-p'],
                        help='Specifies the type of key protection.')
-            c.argument('disabled', arg_type=get_three_state_flag(), help='Create key in disabled state.')
+            c.argument('disabled', arg_type=get_three_state_flag(), help='Create key in disabled state.', default=False)
```

Comment:
By setting `default=False`, CLI will set the default behavior for creating key.
But we want to leave the control to service side. That's why previous default is `None`.

https://github.com/Azure/azure-cli/issues/7479 should be a ask for keyvault service , not for CLI.